scripts/utils/zowe-install-xmem.sh

```diff
@@ -39,40 +53,36 @@ echo "<$SCRIPT>" | tee -a ${LOG_FILE}
 echo started from `pwd` >> ${LOG_FILE}
 
 # check parms
-if [[ $# -lt 3 || $# -gt 4 ]]
+if [[ -z ${data_set_prefix} || -z ${loadlib} || -z ${parmlib} ]]
 then
-echo Expected 3 or 4 parameters, found $# | tee -a ${LOG_FILE}
-echo Parameters supplied were $@ | tee -a ${LOG_FILE}
-echo Usage:
+echo Parameters supplied were $@ >> ${LOG_FILE}
+echo "Not all the required parameters were supplied"
```

Comment:
It would be friendly to tell the user which parm(s) they omitted.

Reply:
Updated to output the missing parameters